Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Builds a Docker to deliver dist/
-FROM node:11.0
+FROM docker.artifactory.acorn.cirrostratus.org/node:11.0
```

Comment:
Not related to Docker change, but confusing that we're using `node:11.0` here, but specify `node:10` in our config https://github.com/ithaka/aiw-ui/blob/stage/.nvmrc

Reply:
good one @NazimAli2017 . Should we use node 10 in Dockerfile too ?

Reply:
Also, this is a public repository.

Reply:
after considering slack conversation https://ithaka.slack.com/archives/CMMR1SR6K/p1605303278003200 , we decided its okay to move forward with pointing to internal artifactory now. Also, for node 12, we are going to have separate ticket to resolve that issue. So resolving this conversation.